packages/hadron-demo/index.ts

```diff
@@ -43,15 +43,17 @@ jsonProvider([`${__dirname}/routing/*`], ['config.js']).then((routes: any) => {
       hadronExpress,
     ],
     config,
-  ).then((container: IContainer) => {
-    expressApp.use((req, res, next) =>
-      res.status(404).json('Request not found.'),
-    );
-    container.register('customValue', 'From Brainhub with ❤️');
+  )
+    .then((container: IContainer) => {
```

Comment:
maybe you could use `async / await` for a better readability

Reply:
I'd rather stick to then/catch. We are using them all over the project. Also, readability is very subjective topic